web/themes/custom/nysenate_theme/src/patterns/global/fonts/_fonts.scss

```diff
@@ -998,8 +998,3779 @@
 	padding-left: 3px;
 }
 
-//FOUNDATION CSS
+/*---------------------------------/
```

Comment:
Is it possible to break this out into its own include, and use `@import` where necessary?